src/PhpSpreadsheet/Reader/Xlsx/ConditionalStyles.php

```diff
@@ -45,6 +45,7 @@ private function readConditionalStyles($xmlSheet)
                         || (string) $cfRule['type'] == Conditional::CONDITION_CONTAINSTEXT
                         || (string) $cfRule['type'] == Conditional::CONDITION_CONTAINSBLANKS
                         || (string) $cfRule['type'] == Conditional::CONDITION_NOTCONTAINSBLANKS
+                        || (string) $cfRule['type'] == Conditional::CONDITION_NOTCONTAINSTEXT
```

Comment:
Scrutinizer thinks this change makes the code too complex, when it didn't think it was too complex before. That will prevent your change from being merged. It's one of the reasons why I recommended the use of a constant class array earlier - you can reduce 6 separate tests in this statement to 1, and Scrutinizer will be happy with that.